source/_patterns/00-atoms/components/definition-list.mustache

```diff
@@ -1,4 +1,4 @@
-<dl class="definition-list{{#inline}} definition-list--inline{{/inline}}">
+<dl class="definition-list{{#inline}} definition-list--inline{{/inline}}{{#timeline}} definition-list--timeline{{/timeline}}">
```

Comment:
This should be 

```
{{#variant}} definition-list--{{variant}}{{/variant}}
```

And we can add `variant` to yaml schema as optional property:

```
  variant:
    type: string
    enum:
      - timeline
```

Reply:
Schema is now updated with enum variant and mustache template adapted.